### PR TITLE
Feature/childcare subsidy

### LINKED ISSUE #26: Childcare subsidy

### PR DESCRIPTION
**Pull request by dlouise64**
Tax and benefit system evolution.

Details:
Adding Childcare Subsidy (closes #26)

### New Act/Reg specially defined variables
 * `childcare_subsidy__is_a_dependant_child`
 * `childcare_subsidy__child_is_no_older_than_5_years_28_days`
 * `childcare_subsidy__is_child_attending_school`
 * `childcare_subsidy__child_will_be_enrolled_to_school_with_cohort_policy`
 * `childcare_subsidy__child_has_disability_allowance`

### CODE REVIEW

**Comment by dlouise64** [comment]
@Br3nda ready for review but can you please let me know which directory these should be stored in. What number should the childcare subsidy directory be. Thanks

File: openfisca_aotearoa/variables/social_security/childcare_subsidy.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
+
+# -*- coding: utf-8 -*-
+
+from openfisca_core.model_api import *
+from openfisca_aotearoa.entities import Person
+
+
+class childcare_subsidy__is_a_dependant_child(Variable):
```
Comment (Br3nda) [inline openfisca_aotearoa/variables/social_security/childcare_subsidy.py]: http://www.legislation.govt.nz/regulation/public/2004/0268/latest/DLM282545.html
is section 12, so would be in a folder called 12

**Comment by Br3nda** [comment]
http://www.legislation.govt.nz/regulation/public/2004/0268/latest/DLM282545.html
is section 12, so would be in a folder called 12

File: openfisca_aotearoa/variables/acts/social_security/12/childcare_subsidy.py [hunk 2]
```diff
@@ -0,0 +1,64 @@
+
+# -*- coding: utf-8 -*-
+
+from openfisca_core.model_api import *
+from openfisca_aotearoa.entities import Person
+
+
+class childcare_subsidy__is_a_dependant_child(Variable):
```
Comment (Br3nda) [inline openfisca_aotearoa/variables/acts/social_security/12/childcare_subsidy.py]: We'd only need a special variable for this if childcare_subsidy has it's only unique defintion of what a dependent child it.

i think it should be the existing dependent child under the interpretation in the social security act.

**Pull request closed without merging**